pkg/storage/unified/resource/resource.proto

```diff
@@ -476,14 +476,11 @@ message ListRepositoryObjectsRequest {
   // Starting from the requested page (other query parameters must match!)
   string next_page_token = 1;
 
-  // All results exist within this resource namespace+name
-  ResourceKey key = 2;
-
+  // Namespace (tenant)
+  string namespace = 2;
+  
   // The name of the repository
-  string name = 3;
-
-  // Maximum number of items to return
-  int64 limit = 4;
+  string repository = 3;
```

Comment:
Is this already used somewhere? If so, we have to make these fields `reserved` and use new tag numbers.